aireview/poetry.lock

```diff
@@ -1,5 +1,188 @@
 # This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
 
```

Comment:
**documentation**: Ensure changes to poetry.lock are accompanied by corresponding updates in pyproject.toml to maintain dependency transparency. The lock file should only be updated through 'poetry update' or 'poetry lock' commands.